docs/book/src/tasks/experimental-features/cluster-class/write-clusterclass.md

```diff
@@ -609,7 +612,8 @@ spec:
 In addition to variables specified in the ClusterClass, the following builtin variables can be 
 referenced in patches:
 - `builtin.cluster.{name,namespace,uid,metadata.labels,metadata.annotations}`
-- `builtin.cluster.topology.{version,class,classNamespace}`
+- `builtin.cluster.topology.{version,classRef.name,classRef.namespace,class,classNamespace}`
+    - Note: `class` and `classNamespace` are deprecated and will be removed with the next apiVersion.
```

Comment:
FYI, similar note added to https://github.com/kubernetes-sigs/cluster-api/pull/12236

Reply:
Thx!